gateway/src/task.rs

```diff
@@ -491,7 +491,7 @@ impl Task<()> for ProjectTask {
 
         let span = info_span!(
             "polling project",
-            ctx.project = ?project_ctx.project_name.to_string(),
+            ctx.project = %project_ctx.project_name,
             ctx.state = project_ctx.state.state(),
             ctx.state_after = field::Empty
```

Comment:
Maybe we should rename these to `shuttle.project.name` or something?

Reply:
We can do that, we currently have a lot of different names for the same thing, e.g. "project", "project_name", "ctx.project".

Reply:
I cleaned this up, so now all references to a project name should have they key `shuttle.project.name`. Tested in honeycomb that there are no traces where the old keys exist after running a bunch of project commands. I also changed `shuttle.service` to ` shuttle.service.name`, and replaced all instances of `service_name` keys with it.